internal/providers/terraform/ibm/ibm.go

```diff
@@ -57,6 +57,7 @@ var globalCatalogServiceId = map[string]catalogMetadata{
 	"ibm_is_flow_log":               {"is.flow-log-collector", []string{}, nil},
 	"ibm_cloudant":                  {"cloudantnosqldb", []string{}, nil},
 	"ibm_pi_instance":               {"power-iaas", []string{}, nil},
+	"ibm_cos_bucket":                {"standard-bucket", []string{}, nil},
```

Comment:
That is not correct. It could be satcon too. 
Probably this is the right object for it:
https://globalcatalog.cloud.ibm.com/update/dff97f5c-bc5e-4455-b470-411c3edbe49c?noLocations=true&tab=overview

Reply:
yes, it could be for satcon too.  The displayname for the cos instances versus the displayname for the buckets should be different, if possible. The COS instances already refer to that GC object you list, but for buckets there's only standard-bucket. Perhaps we list the buckets the same for now until there's something better in GC.

Reply:
I don't think we list cos instances at all, only buckets. And with a satellite cos bucket the name will be confusing IMO. No?
```
resource "ibm_satellite_location" "create_location" {
  location          = var.location
  zones             = var.location_zones
  managed_from      = var.managed_from
  resource_group_id = data.ibm_resource_group.group.id
}

resource "ibm_cos_bucket" "cos_bucket" {
  bucket_name           = "cos-sat-terraform"
  resource_instance_id  = data.ibm_resource_instance.cos_instance.id
  satellite_location_id  = data.ibm_satellite_location.create_location.id
  object_versioning {
    enable  = true
  }
  expire_rule {
    rule_id = "bucket-tf-rule1"
    enable  = false
    days    = 20
    prefix  = "logs/"
  }
}
```

Reply:
I'm not sure if I follow

Reply:
The UX designs call for listing the COS instances as well. The thought is that anything that will get provisioned in an account should be listed. I think it will be somewhat confusing if buckets and instances appear with the same name. And it would also be confusing if sat buckets appears as "standard buckets"...